skins/shared/functions.scss

```diff
@@ -11,3 +11,94 @@
 	}
 	@return nth($range, 2);
 }
+
+@function luminance($color) {
```

Comment:
Alternatives:
1. Download it as dependency, but we don't have package manager
2. Move implementation to JS, but then luminance will be calculated each time on client side
3. Use built-in sass lightness method, which has a different meaning
4. Simplify luminance calculation to: 0.3*Red + 0.59*Green + 0.11*Blue